tf2_geometry_msgs/PoseStamped garage_pose_from_camera
tf2_geometry_msgs/PoseStamped camera_pose_from_garage
float64 angle_to_garage
float64 depth_to_garage